<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWidget</class>
 <widget class="QWidget" name="FormWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*	QWidget	*/
QWidget{
	color: #2c3e50;
	background-color: #ffffff;
}
/*	QLabel	*/
QLabel{
	font-family: Roboto;
	text-align: center;
	font-size: 16px;
}
QLabel#lblTitle{
	font-family: Roboto;
	font-weight: 100;
	font-size: 34px;
}
QLabel#lblIdentifier{
	color: #ffffff;
}
QLabel#lblStatus{
	font-size: 12px;
	color: #ecf0f1;
}
/*	QLineEdit	*/
QLineEdit{
	height: 32px;
	font-family: Roboto;
	font-size: 16px;
	color: #2c3e50;
	border: 0px;
	border-bottom: 1px solid #bdc3c7;
	selection-background-color: #34495e;
	selection-color: #ffffff;
}
/*	QPushButton	*/
QPushButton{
	font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	text-align: center;
	color: #ecf0f1;
	border: 0;
	border-radius: 3px;
	background-color: #8e44ad;
}
QPushButton:hover{
	background: #9b59b6;
}
QPushButton#btnCancel{
	background-color: #2c3e50;
}
QPushButton#btnCancel:hover{
	background-color: #34495e;
}
QPushButton#btnEdit{
	font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	text-align: center;
	color: #ecf0f1;
	border: 0;
	border-radius: 3px;
	background-color: #2980b9;
}
QPushButton#btnEdit:hover{
	background: #3498db;
}
QPushButton#btnRem{
	font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	text-align: center;
	color: #ecf0f1;
	border: 0;
	border-radius: 3px;
	background-color: #c0392b;
}
QPushButton#btnRem:hover{
	background: #e74c3c;
}
/*	QComboBox	*/
QComboBox{
	height: 32px;
	font-family: Roboto;
	font-size: 16px;
	color: #2c3e50;
	border-radius: 4px;
	border: 1px solid #bdc3c7;
}
QComboBox QAbstractItemView {
	selection-background-color: #2c3e50;
	selection-color: #ffffff;
}
QComboBox::drop-down {
	border: 0;
	border-radius: 4px;
}
QComboBox::down-arrow{   
	image: url(resources/icons/arrows-down.png); 
	width: 14px;
	margin-right: 5px;
}
/*	QDateEdit	*/
QDateEdit{
	height: 32px;
	font-family: Roboto;
	font-size: 16px;
	color: #2c3e50;
	border: 1px solid #bdc3c7;
	border-radius: 5px;
	selection-background-color: #2c3e50;
	selection-color: #ffffff;
}
QDateEdit::drop-down{ border: 0; }
QDateEdit::down-arrow{
	image: url(resources/icons/arrows-down.png); 
	width: 14px;
}
/*	QCalendarWidget	*/
QCalendarWidget {
	font-family: Roboto;
	font-size: 12px;
}
QCalendarWidget QAbstractItemView {
	background-color:#ffffff; 
	selection-background-color: #2c3e50;
	selection-color: #ecf0f1;
}
QCalendarWidget QToolButton {
	border: 0;
	border-radius: 5px;
	icon-size: 20px, 20px;
	image: url(resources/icons/arrows-down.png); 
}
QCalendarWidget QToolButton::menu-arrow {
	width: 14px;
	image: url(resources/icons/arrows-left.png); 
	padding: 20px;
}
QCalendarWidget QToolButton::menu-button {
	width: 14px;
	image: url(resources/icons/arrows-right.png); 
	padding: 20px;
}
QCalendarWidget QToolButton::menu-indicator {
	width: 14px;
	image: url(resources/icons/arrows-down.png); 
	padding: 20px;
}
/*	QSpinBox	*/
QSpinBox::up-button { 
	width: 14px;
	image: url(resources/icons/arrows-up.png); 
}
QSpinBox::down-button { 
	width: 14px;
	image: url(resources/icons/arrows-down.png); 
}
/*	ScrollBar	*/
QScrollBar:vertical{
	width: 5px;
}
QScrollBar::handle:vertical {
	min-height: 10px;
	background-color: #2c3e50;
}
QScrollBar::handle:vertical {
	min-height: 10px;
	background-color: #34495e;
	border: 0;
	border-radius: 2px;
}
/*	QTableWidget	*/
QTableWidget{
	height: 32px;
	font-family: Roboto;
	font-size: 16px;
	border: 1px solid #ecf0f1;
	border-radius: 5px;
	color: #000000;
	background-color: #ffffff;
	alternate-background-color: #ecf0f1;
	selection-background-color: #2c3e50;
}
/*	QHeaderView	*/
QHeaderView::section{
	color: #2c3e50;
	border: 1px solid #bdc3c7;
	border-radius: 1px;
	background-color: #ffffff;
	font-family: Roboto;
	text-align: center;
	font-size:16px;
	height: 32px;
}
/* AboutWidget */
QWidget#AboutWidget QLabel{
	font-family: Roboto;
	font-weight: 100;
	text-align: center;
	font-size: 16px;
}
QWidget#AboutWidget QLabel#lblTitle{
	font-family: Roboto;
	font-weight: 100;
	font-size: 100px;
}
/* QMessageBox */
QMessageBox{
	background-color: #2c3e50;
}
QMessageBox QLabel{
	background-color: #2c3e50;
	color: #ecf0f1;
	font-family: Roboto;
	font-size: 14px;
}
QMessageBox QPushButton{
	border: 0;
	border-radius: 3px;
	background-color: #8e44ad;
	color: #ecf0f1;
    font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	min-height: 30px;
	min-width: 120px;
}
QMessageBox QPushButton:hover{
	background-color: #9b59b6;
}
QMessageBox QPushButton#btnInformative{
	border: 0;
	border-radius: 3px;
	background-color: #2980b9;
	color: #ecf0f1;
    font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	min-height: 30px;
	min-width: 120px;
}
QMessageBox QPushButton#btnInformative:hover{
	background-color: #3498db;
}
QMessageBox QPushButton#btnWarning{
	border: 0;
	border-radius: 3px;
	background-color: #c0392b;
	color: #ecf0f1;
    font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	min-height: 30px;
	min-width: 120px;
}
QMessageBox QPushButton#btnWarning :hover{
	background-color: #e74c3c;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>Cadastro de Cliente</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlStatus">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="hsLeft_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsRight_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlCenter">
     <item>
      <spacer name="hsLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frmContent">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="lblPersonalData">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Dados Pessoais:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtFullname">
          <property name="maxLength">
           <number>255</number>
          </property>
          <property name="placeholderText">
           <string>Nome completo</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_1">
          <item>
           <widget class="QLineEdit" name="txtCpf">
            <property name="maxLength">
             <number>14</number>
            </property>
            <property name="placeholderText">
             <string>CPF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPhone">
            <property name="maxLength">
             <number>15</number>
            </property>
            <property name="placeholderText">
             <string>Telefone</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_2">
          <item>
           <widget class="QComboBox" name="cbGender">
            <item>
             <property name="text">
              <string>Masculino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Feminino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Outro</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtBirthday">
            <property name="currentSection">
             <enum>QDateTimeEdit::DaySection</enum>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="vsMiddle_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblAddress">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Endereço:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtStreet">
          <property name="maxLength">
           <number>255</number>
          </property>
          <property name="placeholderText">
           <string>Logradouro</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_3">
          <item>
           <widget class="QLineEdit" name="txtNumber">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>Número</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtComplement">
            <property name="minimumSize">
             <size>
              <width>225</width>
              <height>0</height>
             </size>
            </property>
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>Complemento</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_4">
          <item>
           <widget class="QLineEdit" name="txtDistrict">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>Bairro</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbState">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>UF</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>UF</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbCity">
            <property name="minimumSize">
             <size>
              <width>175</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Cidade</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="vsMiddle_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>SALVAR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>CANCELAR</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="hsRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblIdentifier">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtFullname</tabstop>
  <tabstop>txtCpf</tabstop>
  <tabstop>txtPhone</tabstop>
  <tabstop>cbGender</tabstop>
  <tabstop>dtBirthday</tabstop>
  <tabstop>txtStreet</tabstop>
  <tabstop>txtNumber</tabstop>
  <tabstop>txtComplement</tabstop>
  <tabstop>txtDistrict</tabstop>
  <tabstop>cbState</tabstop>
  <tabstop>cbCity</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
